launcher/Application.cpp

```diff
@@ -1,8 +1,9 @@
 // SPDX-License-Identifier: GPL-3.0-only
```

Comment:
temporary conversation because of the accidental Scrumplex approval :p

_hmm yes very interesting SPDX thing_ :monocle_face:

Reply:
lol

Reply:
yes, more conversations please.
![image](https://user-images.githubusercontent.com/31988415/200086425-9e51e2aa-3bde-489b-a859-a425b5e66e8e.png)